sdk/storage/storage-file-datalake/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Release History
 
-## 12.0.1 (Unreleased)
+## 12.0.1 (2020.05)
```

Comment:
It has not been clearly decided whether the May update is for all storage libraries or only for blob.
@jiacfan Can you provide clarification on this?

@jeremymeng, If only blob is being released in May, then the above date update needs to be reverted for other storage packages

Reply:
Yes if we just target the tracing issue then only blob.

Reply:
@jiacfan Waiting on you for clarification.